index.bs

```diff
@@ -843,4 +1380,227 @@ result = (
   unknown-error: 199
 )
 
+; type key 17
+remote-playback-availability-request = {
+  request
+  1: [* text] ; urls
+  2: microseconds ; watch-duration
+  3: int ; watch-id
+}
+
+; type key 18
+remote-playback-availability-response = {
+  response
+  url-availabilities: [* url-availability]
+}
+
+; type key 114
+remote-playback-availability-event = {
+  1: int ; watch-id
+  2: [* url-availability] ; url-availabilities
+}
+
+; type key 115
+remote-playback-start-request = {
+  request
+  1: remote-playback-id ; remote-playback-id
+  2: [* text] ; urls
+  ? 3: [* text] ; text-track-urls
+  ? 4: [* http-header] ; headers
+  ? 5: remote-playback-controls ; controls
+}
+
+; type key 116
+remote-playback-start-response = {
+  response
+  1: &result ; result
+  ? 2: remote-playback-state ; state
+}
+
+; type key 117
+remote-playback-termination-request = {
+  request
+  1: remote-playback-id ; remote-playback-id
+  2: &(
+    user-terminated-via-controller: 11
+    unknown: 255
+  ) ; reason
+}
+
+; type key 118
+remote-playback-termination-response = {
+  response
+  1: &result ; result
+}
+
+; type key 119
+remote-playback-termination-event = {
```

Comment:
Do we still need ` 0: request-id ; request-id` added here, even if isn't a request or response?